### ISSUE TITLE
Apple's beta review.

### ISSUE THREAD

**Issue by SzymonMatysik** [ittybittyapps/appstoreconnect-cli#247]
Hey, i'm looking for this feature:
[Submit an app for beta review](https://developer.apple.com/documentation/appstoreconnectapi/submit_an_app_for_beta_review)
am i missing something or it's not implemented yet?

**Comment by orj** [comment]
Yeah that is not yet implemented sorry. PRs and contributions appreciated. :)

**Comment by orj** [comment]
BTW if you're looking for the ability to upload beta builds to App Store Connect you can just use the altool command line tool which is bundled with Xcode.

You only need to submit a beta app review once after an upload.

We definitely want to add support for this to the CLI but haven't got to it yet.
